test/core/pthread/emscripten_futexes.c

```diff
@@ -2,20 +2,21 @@
 #include <emscripten/threading.h>
 #include <errno.h>
 #include <limits.h>
+#include <math.h>
 #include <stdio.h>
 
 int main() {
   // This should return -EWOULDBLOCK.
   int futex_value = 42;
-  int rc = emscripten_futex_wait(&futex_value, futex_value + 1, 0);
+  int rc = emscripten_futex_wait(&futex_value, futex_value + 1, INFINITY);
```

Comment:
Should we not also check for a timeout of 0?

Reply:
(I mean, have a test for a timeout of 0)

Reply:
actually I think I can just revert this file..